### PR TITLE
[WIP] Feature/main page 27

### PR DESCRIPTION
**Pull request by Merciful12**
https://github.com/sicpa-dlab/didcomm-gemini/issues/27

implement base project structure
implement main page

### CODE REVIEW

File: site/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "trailingComma": "all",
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true,
+  "printWidth": 140,
+  "bracketSpacing": true,
+  "arrowParens": "always"
```
Comment (asmarty) [inline site/.prettierrc]: Please add `"endOfLine": "auto"` to avoid end line issue for different OS

File: site/gatsby-node.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+const path = require('path')
+
+module.exports.onCreateNode = ({ node, actions, getNode }) => {
+  if (node.internal.type === 'MarkdownRemark') {
+    const fileNode = getNode(node.parent)
+    const parsedFilePath = path.parse(fileNode.relativePath)
+    const { sourceInstanceName } = fileNode
+
+    if (sourceInstanceName === 'protocols') {
```
Comment (asmarty) [inline site/gatsby-node.js:9]: Can we get rid of nested If?

**Review by asmarty — APPROVED** [review]

File: site/gatsby-node.js (hunk 2 above)
Comment (Merciful12) [reply site/gatsby-node.js:9]: yes as for now, but in future there might be `sourceInstanceName == pages`  for example

File: site/.prettierrc (hunk 1 above)
Comment (Merciful12) [reply site/.prettierrc]: fixed

**Pull request merged**